prow/config.yaml

```diff
@@ -9112,6 +9112,23 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- interval: 6h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gce-coredns-performance
+  labels:
+    preset-service-account: true
+    preset-k8s-ssh: true
+  spec:
+    containers:
+    - args:
+      - --timeout=11
```

Comment:
This is kind of a higher-level timeout and I think should be at least 20m more that the job timeout (the one you specified in config.json).